RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate Database to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce


Test cleaning the cache, new messages that are coming when the app is in foreground and background, edit users profiles and room info, create new threads, talk in other threads, use the search for threads (`ThreadMessagesView`) and clean again the cache. The data should be the same on the web. 

## Screenshots

## Types of changes




## Checklist



## Further comments
